src/components/tutor-input.cjsx

```diff
@@ -141,7 +141,7 @@ TutorDateInput = React.createClass
           weekStart={0}
         />
     else if @props.disabled and value
-      displayValue = value.toString("YYYY/MM/DD")
+      displayValue = value.toString("MM/DD/YYYY")
 
```

Comment:
Could "MM/DD/YYYY" be set as a default prop or something throughout to help with potential future date formatting changes?